src/main/scala/org/apache/spark/streamdm/streamDMJob.scala

```diff
@@ -33,13 +34,24 @@ object streamDMJob {
 
     //configuration and initialization of model
     val conf = new SparkConf().setAppName("streamDM")
-    conf.setMaster("local[2]")
 
-    val ssc = new StreamingContext(conf, Seconds(10))
+    var newArgs = args.clone()
+    var batchInterval: Int = 1000
+    if(args.length > 0){
+      if(Try(args(0).toInt).isSuccess){
+        if(args(0).toInt > 0 && args(0).toInt < Int.MaxValue){
```

Comment:
Too many args(0).toInt

Reply:
@nhnminh You can create an intermediary variable to avoid calling `args(0).toInt` too many times

Reply:
Hi @zhangjiajin  and @hmgomes, I created an intermediate variable, so-called firstArg for this args(0) as you suggested.